### PR TITLE
feat(TDP-3460/backend): add dq ui access rights management

### PR DESCRIPTION
**Pull request by bdiouf**
**Link to the JIRA issue**
e.g. https://jira.talendforge.org/browse/TDP-3460

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)

**(Optional) Other information**:

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/123/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/123/)

File: dataprep-api/src/main/java/org/talend/dataprep/api/service/settings/AppSettingsConfigurer.java [hunk 1]
```diff
@@ -51,6 +51,9 @@
      */
     public abstract T configure(final T setting);
 
+
+    public boolean isUserAuthorized(){return true;}
```
Comment (fhuaulme) [inline dataprep-api/src/main/java/org/talend/dataprep/api/service/settings/AppSettingsConfigurer.java]: Code format.

Comment (bdiouf) [inline dataprep-api/src/main/java/org/talend/dataprep/api/service/settings/AppSettingsConfigurer.java]: Thank you

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/125/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/125/)

**Pull request merged**